chat/src/Home.js

```diff
@@ -0,0 +1,38 @@
+import { Link } from "react-router-dom";
+import {React} from "react";
+import { Navbar } from "react-bootstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+const Home = () => {
+  return (
+    <>
+      <Navbar className="container-fluid nav-bar">
+        <Link
+          to="/"
+          className=" text-black text-decoration-none"
+          style={{ margin: "10px" }}
+        >
+          <img src="logo192.png" alt="logo" className="nav-logo" />
+        </Link>
+        <div className="nav-text">
+          <Link
+            to="/Login"
+            className="nav-login"
+            style={{ margin: "20px" }}
```

Comment:
There are built-in classes for margin in bootstrap. You can used that here or create a class for it.